scrapy/shell.py

```diff
@@ -115,6 +115,9 @@ def fetch(self, request_or_url, spider=None):
         self.populate_vars(response, request, spider)
 
     def populate_vars(self, response=None, request=None, spider=None):
+        import scrapy
+
+        self.vars['scrapy'] = scrapy
```

Comment:
I think it worths mentioning in get_help, something like "scrapy is already imported"